ci/playbooks/multinode-customizations.yml

```diff
@@ -64,10 +64,16 @@
         state: restarted
         name: crc-pre
 
-    - name: Restart dnsmasq
```

Comment:
If this is finally enough I'd drop a comment here, in the code, to avoid someone in the future to break this by changing this back to a restart.

Reply:
Yeah, it seems to work better now. Added a comment. Tks

Reply:
I had to add an additional check if crc-dnsmasq is absent, before starting again. Seems more stable now. Testproject ended with success 3x in a row:
https://review.rdoproject.org/r/c/testproject/+/48840